.gitignore

```diff
@@ -7,3 +7,5 @@ artifacts/@openzeppelin
 artifacts/build-info
 artifacts/contracts/**/**.dbg.json
 .openzeppelin/unknown-298.json
+.env
+test-results.*
```

Comment:
we tend to include a blank line at the end of any file

Reply:
Ahh I see! Will add it now!